cmd/brew-bootstrap-asdf

```diff
@@ -48,7 +48,13 @@ EOF
   bash ~/.asdf/plugins/nodejs/bin/import-release-team-keyring
 fi
 
-asdf install
+(
+  for var_override in $( env | grep -E '^ASDF_INSTALL_VAR_[^=]+=' ); do
+    var="${var_override#ASDF_INSTALL_VAR_}"
+    export "${var}=${!var_override@Q}"
```

Comment:
These lines are converting a variable like `ASDF_INSTALL_VAR_X=<value of X>` to `X=<value of X>`, using some more complex bash parameter expansions. See "Parameter Expansion" in https://manpages.org/bash#parameters

The first line uses the "Remove matching prefix pattern" `${parameter#word}` syntax to convert "ASDF_INSTALL_VAR_X" into "X".

The second line runs an export statement, to set the variable of that name. Normally an export would be like `VAR="...something..."` with a literal string on the left side of the =. But the whole VAR=something bit can be a quoted string (like any other command argument), and the left side of the equals is allowed to use a variable expansion as well.

The right hand side of that line sets the new variable (e.g. X) to the value of the old one (e.g. ASDF_INSTALL_VAR_X). `${var_override}` would evaluate to the old var's name "ASDF_INSTALL_VAR_X". `${!var_override}"` with the exclamation mark instead uses indirection to look up the value of the variable with that name. The `@Q` on the end puts the resulting variable value in quotes; I'm not sure if this is necessary but it seems to work OK and feels a little safer.